app/uk/gov/hmrc/serviceconfigs/model/FrontendRoutes.scala

```diff
@@ -65,19 +67,22 @@ object FrontendRoutes {
     FrontendRoutes(
       environment = mfr.environment,
       service     = mfr.service,
+      routesFile  = mfr.routesFile,
       routes      = Seq(FrontendRoute.fromMongo(mfr)))
 
   def fromMongo(mfrs: Seq[MongoFrontendRoute]): Seq[FrontendRoutes] =
-    mfrs.groupBy(frs => (frs.service, frs.environment))
+    mfrs.groupBy(frs => (frs.service, frs.environment, frs.routesFile))
```

Comment:
could be more manageable to populate FrontendRoutes with routes = Seq.empty, rather than a 3-tuple?